aiinfra-cluster/main.tf

```diff
@@ -162,8 +174,11 @@ module "aiinfra-compute" {
   depends_on = [
     module.aiinfra-network
   ]
-  enable_gke = var.orchestrator_type == "gke"
-  gke_version = var.gke_version
+  orchestrator_type            = var.orchestrator_type
+  slurm_node_count_static      = var.slurm_node_count_static
+  slurm_node_count_dynamic_max = var.slurm_node_count_dynamic_max
+  network_storage              = flatten([module.nfs_filestore[*].network_storage])
```

Comment:
we should pass GCS buckets as well, right ?

Reply:
for sure, just added it

Reply:
Have you tested it with both GCSFuse and NFS filestore locally ?

Reply:
yeah I messed that up big time but just fixed it

Reply:
thanks for testing it.